src/neptune_scale/net/util.py

```diff
@@ -0,0 +1,6 @@
+def escape_nql_criterion(criterion: str) -> str:
+    """
+    Escape backslash and (double-)quotes in the string, to match what the NQL engine expects.
+    """
+
+    return criterion.replace("\\", r"\\").replace('"', r"\"")
```

Comment:
```suggestion
    return criterion.replace(r"\", r"\\").replace('"', r"\"")
```
for a better clarity